### ISSUE TITLE
code ELIFECYCLE when trying to install with npm

### ISSUE THREAD

**Issue by nicolasreszka** [liteflow-labs/liteflow-js#218]
On macOs Sierra 10.12.6 (16G2136)

```
macbook-pro-de-macbook:liteflow-test macbook$ npm install -g @liteflow/cli
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN checkPermissions Missing write access to /usr/local/lib/node_modules
npm ERR! code EACCES
npm ERR! syscall access
npm ERR! path /usr/local/lib/node_modules
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, access '/usr/local/lib/node_modules'
npm ERR!  [Error: EACCES: permission denied, access '/usr/local/lib/node_modules'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/local/lib/node_modules'
npm ERR! }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/macbook/.npm/_logs/2020-08-20T11_37_56_344Z-debug.log
macbook-pro-de-macbook:liteflow-test macbook$ sudo npm install -g @liteflow/cli
Password:
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
/usr/local/bin/liteflow -> /usr/local/lib/node_modules/@liteflow/cli/bin/run

> iso-constants@0.1.2 install /usr/local/lib/node_modules/@liteflow/cli/node_modules/iso-constants
> node build.js > index.browser.js

sh: index.browser.js: Permission denied
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! iso-constants@0.1.2 install: `node build.js > index.browser.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the iso-constants@0.1.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/macbook/.npm/_logs/2020-08-20T11_38_38_156Z-debug.log
macbook-pro-de-macbook:liteflow-test macbook$ node -v
v12.18.3
```

**Comment by antho1404** [comment]
Thanks for the logs, this seems related to https://github.com/liteflow-labs/liteflow-js/issues/19

I will add a bit more context to help reproduce based on our discussion.

This is based on an install of node using the bundler in the https://nodejs.org/en/ website with node v12.18.3 and the issue also occurs when running the command with `sudo`.

I think this is due to the installer that creates a different environment than the one created with `nvm`.

**Comment by nicolasreszka** [comment]
Here is the log file. 
[2020-08-20T11_38_38_156Z-debug.log](https://github.com/liteflow-labs/liteflow-js/files/5103013/2020-08-20T11_38_38_156Z-debug.log)

**Comment by antho1404** [comment]
A new version of the CLI has been released ([2.4.0](https://www.npmjs.com/package/@liteflow/cli)) and it solves this issue.

You should now be able to install the CLI without the flag `unsafe-perm`. 

If you install node with the installer you will still need to install it in global with sudo.

Please let me know if you still have the issue.

**Comment by nicolasreszka** [comment]
Installation worked fine. Thank you! 
![Capture d’écran 2020-08-25 à 13 30 22](https://user-images.githubusercontent.com/9148582/91169267-519fb680-e6d7-11ea-8b76-b96f9bd15c0e.jpg)
